src/apis/chat.ts

```diff
@@ -0,0 +1,18 @@
+import { client } from '@/apis';
+
+const CHAT_URL = '/v1/api/chat';
+
+export const prevChat = async (customRoomId: string | null) => {
+  const response = await client.get(`${CHAT_URL}/detail/${customRoomId}`);
+  return response;
+};
+
+export const userChat = async (sellerId: number) => {
+  const response = await client.get(`${CHAT_URL}/user/${sellerId}`);
+  return response;
+};
+
+export const sellerChat = async (sellerId: number, productId: number) => {
+  const response = await client.get(`${CHAT_URL}/user/${sellerId}/${productId}`);
+  return response;
+};
```

Comment:
해당 API 사용하는 곳들 보니깐 response 가 아닌 response.data 의 값을 이용하는 것으로 보입니다.

```suggestion
export const sellerChat = async (sellerId: number, productId: number): Promise<반환값> => {
  const response = await client.get(`${CHAT_URL}/user/${sellerId}/${productId}`);
  return response.data;
};
```
위와  같은 형태로 하는건 어떤가요?